plugins/doc_fragments/foreman.py

```diff
@@ -325,3 +325,35 @@ class ModuleDocFragment(object):
     type: str
     required: False
 '''
+
+    INFOMODULE = '''
+options:
+  name:
+    description:
+      - Name of the resource to fetch information for.
+      - Mutually exclusive with I(search).
+    required: false
+    type: str
+  location:
```

Comment:
Do all resourced have taxonomies?
Well probably all that we have infomodules for at this point. Which is good enough for now.

Reply:
Yeah, `setting` is not taxonomied, but I've left that to future-me to fix :)